lalrpop/src/lexer/re/mod.rs

```diff
@@ -19,6 +19,11 @@ pub fn parse_literal(s: &str) -> Regex {
 
 /// Parse a regular expression like `a+` etc.
 pub fn parse_regex(s: &str) -> Result<Regex, RegexError> {
-    let expr = Parser::new().parse(s)?;
+    let enable_unicode = cfg!(feature = "unicode");
+    let expr = ParserBuilder::new()
+        .utf8(enable_unicode)
```

Comment:
Shouldn't we always require regexp to match only utf8 tokens? And even if we want to allow, should this option really be tied to unicode support?